src/sim_recon/images/dv.py

```diff
@@ -75,25 +75,68 @@ def get_dv_axis_sizes(dv: mrc.Mrc) -> dict[str, int]:
 
 
 def get_wavelengths_from_dv(dv: mrc.Mrc) -> Generator[Wavelengths, None, None]:
+
+    def handle_rec_array(
+        ext_floats: np.ndarray[Any, np.dtype[np.record]],
+        wavelength_index: int,
+        header_shape: tuple[int, ...],
+    ) -> Generator[Wavelengths, None, None]:
+        # Reshape header array to match the image ordering so we only have to
+        # check each defined wavelength
+        ext_floats = ext_floats.reshape(*header_shape)
+
+        for c in range(header_shape[wavelength_index]):
+            indexes = [0, 0, 0]
+            indexes[wavelength_index] = c
+            yield Wavelengths(
+                # stored as a recarray of records, so getting the records need
+                # to be indexed separately
+                ext_floats[*indexes]["exWavelen"],
```

Comment:
Does this `[*indexes]` work? Note, using a list triggers fancy indexing where as using a tuple doesn't

Reply:
Since Python 3.11 due to [PEP646](https://peps.python.org/pep-0646/#change-1-star-expressions-in-indexes)

Reply:
I can change to using `tuple(indexes)` but the minimum Python version is set to 3.12 and I do think this is cleaner.

Reply:
It's fine as is - I haven't used 3.11+ as yet...